### PR TITLE
Implement transferserver command! #WIP

### PR DESCRIPTION
**Pull request by NycuRO**

### CODE REVIEW

**Comment by SuperMaXAleX** [comment]
I recommand to add the possibility to choose the player on arg1 you want to transfer for example

**Comment by NycuRO** [comment]
@SuperMaXAleX No ,because that is default command. You can create costum plugin with command if you  like change.

File: src/main/java/cn/nukkit/command/defaults/TransferServerComand.java [hunk 1]
```diff
@@ -3,11 +3,12 @@
 import cn.nukkit.command.CommandSender;
 import cn.nukkit.command.ConsoleCommandSender;
 import cn.nukkit.command.data.CommandParameter;
+import cn.nukkit.event.player.PlayerTransferEvent;
 import cn.nukkit.lang.TranslationContainer;
 import cn.nukkit.Player;
 import cn.nukkit.utils.TextFormat;
 
-public class TransferServerCommand extends VanillaCommand {
+class TransferServerCommand extends VanillaCommand {
```
Comment (NycuRO) [inline src/main/java/cn/nukkit/command/defaults/TransferServerComand.java]: [ERROR] /home/travis/build/Nukkit/Nukkit/src/main/java/cn/nukkit/command/defaults/TransferServerComand.java:[10,8] class TransferServerCommand is public, should be declared in a file named TransferServerCommand.java


Why?

**Comment by NycuRO** [comment]
I don't know how fix errors. If anyone can help me ,ok. If not, close PR.

**Comment by solo5star** [comment]
I found a typo
TransferServerComand.java -> TransferServerCommand.java

**Comment by NycuRO** [comment]
@solo5star thanks

**Comment by imjack** [comment]
This shouldn't be a default command.

**Comment by MrPowerGamerBR** [comment]
@imjack yeah, I don't know why this would be a good command, just because it is a vanilla command in single player doesn't mean it should be a vanilla command in multiplayer.

Or at least make the command available with a config switch (like how /gc and /status work)

**Comment by NycuRO** [comment]
@imjack Is in 1.0.3

**Comment by Snake1999** [comment]
`All checks have failed`

**Comment by AnkitM252** [comment]
I made the new one https://github.com/Nukkit/Nukkit/pull/1502

**Pull request closed without merging**